### PR TITLE
zapier: add page

### PR DESCRIPTION
**Pull request by Sadeeed**
I'm not sure if we should have a generic page for the command but I have made it just in case

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 15.16.1**

#14172

### CODE REVIEW

File: pages/common/zapier.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# zapier
+
+> Create, automate, and manage zapier integrations.
```
Comment (spageektti) [inline pages/common/zapier.md:3]:
```suggestion
> Create, automate, and manage zapier integrations.
> Some subcommands such as `build` have their own usage documentation.
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md

**Review by spageektti — APPROVED** [review]
Thanks for your contribution!

File: pages/common/zapier.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+> Some subcommands such as `build`, `init`, `scaffold`, `push`, `test`, etc. have their own usage documentation.
+> More information: <https://github.com/zapier/zapier-platform/blob/main/packages/cli/docs/cli.md>.
+
+- Connect to your Zapier account:
```
Comment (spageektti) [inline pages/common/zapier.md:7]:
```suggestion
- Connect to an Zapier account:
```

File: pages/common/zapier.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Initialize a new Zapier integration with a project template:
+
+`zapier init {{path/to/directory}}`
+
+- Add a starting trigger, create, search, or resource to your integration:
+
+`zapier scaffold {{trigger|create|search|resource}} {{name}}`
+
+- Test your integration:
```
Comment (spageektti) [inline pages/common/zapier.md:19]:
```suggestion
- Test a integration:
```

File: pages/common/zapier.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`zapier test`
+
+- Build and upload your integration to Zapier:
```
Comment (spageektti) [inline pages/common/zapier.md:23]:
```suggestion
- Build and upload a integration to Zapier:
```

**Review by spageektti — APPROVED** [review]

**Review by spageektti — CHANGES_REQUESTED** [review]
LGTM after my suggestions resolved

File: pages/common/zapier.md (hunk 2 above)
Comment (Sadeeed) [reply pages/common/zapier.md:7]: Should be `a Zapier account` instead of an but good catch

**Review by spageektti — APPROVED** [review]

**Pull request merged**